index.js

```diff
@@ -158,3 +160,83 @@ bot.onText(/\/meetups/, async msg => {
   );
   bot.sendMessage(msg.chat.id, out);
 });
+
+//Feature to save links and articles
+mongoose.connect("mongodb://localhost:27017/saved");
+const savedSchema = new Schema({
+  ChatID: Number,
+  name: String,
+  content: String
+});
+
+const Saved = mongoose.model("Saved", savedSchema);
+
+bot.on("message", async msg => {
+  //getting message string
+  let args = msg.text.split(" ");
+  const chatId = msg.chat.id;
+
+  //Save a note
+  if (args[0] == "/save") {
+    //Check if note with the same name already exists
+    await Saved.find({ ChatID: chatId, name: args[1] }, async (err, notes) => {
+      if (notes.length == 0) {
+        await Saved.create({
+          ChatID: chatId,
+          name: args[1],
+          content: args.splice(2, args.length).join(" ")
+        });
+        bot.sendMessage(chatId, "Note Saved");
+      } else {
+        bot.sendMessage(chatId, "Note already exists");
```

Comment:
Change the error message to point that a `Note with the same name already exists` since the content of the note may differ.